template/package.json

```diff
@@ -63,7 +62,7 @@
     "karma-sinon-chai": "^1.2.0",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-spec-reporter": "0.0.26",
-    "karma-webpack": "^1.7.0",
+    "karma-webpack": "^2.2.0",
```

Comment:
Isn't the latest release of karma-wepack 2.0.2 @limichange?
https://www.npmjs.com/package/karma-webpack